joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/TypeDeclTests.scala

```diff
@@ -449,12 +449,12 @@ class TypeDeclTests extends PhpCode2CpgFixture {
     "generate __construct calls" in {
       inside(cpg.call.name("__construct").l) {
         case constructAnonClass0 :: constructAnonClass1 :: Nil =>
-          constructAnonClass0.code shouldBe "Test0.php:<global>@anon-class-0->__construct(10)"
+          constructAnonClass0.code shouldBe "Test0.php:<global>@anon-class-0.__construct(10)"
```

Comment:
Can you use a `.` instead of the `@` please. Or is there any reference in the PHP world that anonymous classes are `@` prefixed?

Reply:
Changes made.

There is nothing specific about it being an `@`, I was just following the naming pattern on the name changes to go lambdas on codescience.